### ISSUE TITLE
'Usage' examples do not work

### ISSUE THREAD

**Issue by akorb** [secdev/scapy#2494]
https://scapy.readthedocs.io/en/latest/usage.html

Many code examples use lambdas.
With the syntax: 
```python
lambda (s,r):
```

This does not work. There should be no parantheses around the parameters:

```python
>>> add = lambda x, y: x + y
>>> add = lambda (x, y): x + y
  File "<stdin>", line 1
    add = lambda (x, y): x + y
                 ^
SyntaxError: invalid syntax
```

Am I missing something?

**Comment by guedou** [comment]
It is likely that most of the examples only work with Python2. While porting to Python3, we documented lambdas in https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md writing that in Scapy source code `lambdas must be written using a single argument when using tuples: use lambda x, y: x + f(y) instead of lambda (x, y): x + f(y).`.

**Comment by guedou** [comment]
The documentation was fixed in #2508
